src/ShallowTraversal.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import isEmpty from 'lodash/isEmpty';
 import isSubset from 'is-subset';
-import functionName from 'function.prototype.name';
```

Comment:
i think perhaps you included some unintentional changes - have you rebased on the latest master?